New CEF-Related Build Errors in Linux

There are a few new build errors in linux when building with gcc13 and clang, which are a result of the changes for GPU CEF acceleration (11bb8e7).

A few of the errors:

```
modules/webbrowser/src/webrenderhandler.cpp:165:60: error: ‘const class CefAcceleratedPaintInfo’ has no member named ‘shared_texture_handle’
  165 |         memObj, size, GL_HANDLE_TYPE_D3D11_IMAGE_EXT, info.shared_texture_handle
```

```
modules/webbrowser/ext/cef/cef_binary_127.3.5+g114ea2a+chromium-127.0.6533.120_linux64/include/base/cef_ref_counted.h:487:42: error: expected ‘)’ before ‘,’ token
  487 |   explicit RefCountedData(std::in_place_t - , Args&&... args)
```

--- a/modules/webbrowser/cmake/patch/cmake/cef_variables.cmake
+++ b/modules/webbrowser/cmake/patch/cmake/cef_variables.cmake
@@ -103,7 +103,6 @@ if(OS_LINUX)
     -fno-rtti                       # Disable real-time type information
     -fno-threadsafe-statics         # Don't generate thread-safe statics
     -fvisibility-inlines-hidden     # Give hidden visibility to inlined class member functions
-    -std=gnu++14                    # Use the C++14 language standard including GNU extensions
     -Wsign-compare                  # Warn about mixed signed/unsigned type comparisons
     )
   list(APPEND CEF_COMPILER_FLAGS_DEBUG

--- a/modules/webbrowser/include/webrenderhandler.h
+++ b/modules/webbrowser/include/webrenderhandler.h
@@ -67,11 +67,13 @@ public:
     void OnPaint(CefRefPtr<CefBrowser> browser, PaintElementType type,
         const RectList &dirtyRects, const void* buffer, int width, int height) override;
 
+#ifdef WIN32
     // Used when the "shared_texture" flag is set to true for CEF. Uses a shared texture
     // from CEF that is allocated on another part of the GPU. Skip CPU allocationn for
     // better performance. Needs OpenGl 4.5 or higher. 
     void OnAcceleratedPaint(CefRefPtr<CefBrowser> browser, PaintElementType type,
         const RectList& dirtyRects, const CefAcceleratedPaintInfo& info) override;
+#endif // WIN32
 
     // Determines if the alpha value is > 0 at the specified pixel coordinates. Used in
     // the GUI to determine if the click is consumed

--- a/modules/webbrowser/src/webrenderhandler.cpp
+++ b/modules/webbrowser/src/webrenderhandler.cpp
@@ -116,6 +116,7 @@ void WebRenderHandler::OnPaint(CefRefPtr<CefBrowser>, CefRenderHandler::PaintEle
     _needsRepaint = false;
 }
 
+#ifdef WIN32
 void WebRenderHandler::OnAcceleratedPaint(CefRefPtr<CefBrowser> browser,
                                           PaintElementType type,
                                           const RectList& dirtyRects,
@@ -177,6 +178,7 @@ void WebRenderHandler::OnAcceleratedPaint(CefRefPtr<CefBrowser> browser,
     _texture = sharedTexture;
     _needsRepaint = false;
 }
+#endif // WIN32
 
 void WebRenderHandler::updateTexture() {
     if (_acceleratedRendering || _needsRepaint) {

